src/content/1.7/development/components/router-dispatcher.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Router and Dispatcher component
+menuTitle: Router and Dispatcher
+---
+
+# Router and Dispatcher
+
+The legacy Dispatcher and the Symfony Router are in charge of parsing incoming HTTP requests and finding the right Controller to return HTTP response.
+
+### Front office routing
+
+The Front office controllers lie in directory `controllers/front`.
+
+The Dispatcher match a given HTTP request with a Front Office controller.
+
+When you visit `/` on your shop you are being returned the Homepage.
+
+What happened behind the scenes is that
+
+1. Your HTTP request hit the `index.php` main routing file that relies on the _Dispatcher_
+2. The Dispatcher found the right Front controller - the `IndexController`
+3. The `IndexController` calls the `index` Smarty view to be rendered, populated with PHP objects
+
+If you send another HTTP request, the same sequence will happen
+- Dispatcher finds the right Front controller
+- Front controller builds the right response using a Smarty template and relying on PrestaShop business logic
+
+## Back office routing
+
+The Back Office is split between the legacy part and the migrated part.
+
+The legacy backend relies on controllers you can find in directory `controllers/admin`.
+
+The Symfony backend relies on controllers you can find in `src/PrestaShopBundle/Controller/Admin`.
+
+The Back Offie routing match a given HTTP request with a Controller.
+
+### Legacy Back Office routing
+
+When you visit `/admin{x}/index.php?controller=AdminCarriers&token={y}` on your shop you are being returned the Carriers Back Office page.
```

Comment:
```suggestion
When you visit `/admin{x}/index.php?controller=AdminCarriers&token={y}` on your shop you are being returned to the Carriers Back Office page.
```

Reply:
We return a HTML page, not `to a HTML page` ? 🤔 

I think my statement is confusing.
Maybe
```suggestion
When you visit `/admin{x}/index.php?controller=AdminCarriers&token={y}` on your shop, PrestaShop returns the Carriers Back Office HTML page.
```

Reply:
yep, sounds better